core/src/main/java/com/linecorp/armeria/server/FallbackService.java

```diff
@@ -77,7 +84,16 @@ private static HttpResponse handleNotFound(ServiceRequestContext ctx, RoutingCon
                                               .build());
     }
 
-    private static HttpResponse newHeadersOnlyResponse(HttpStatus status) {
+    private static HttpResponse newFallbackResponse(ServiceRequestContext ctx, HttpStatus status) {
+        final ServiceErrorHandler errorHandler = ctx.config().errorHandler();
+        final AggregatedHttpResponse rendered = errorHandler.renderStatus(ctx.config(),
+                                                                          ctx.request().headers(),
+                                                                          status,
+                                                                          null,
+                                                                          null);
+        if (rendered != null) {
+            return rendered.toHttpResponse();
+        }
         // Send a headers-only response as a workaround for the following issue:
```

Comment:
Can you move this comment to L51?   I think this comment now applies to why `exchangeType` aggregates responses in `FallbackService`

Reply:
It looks like we need to rephrase the comment because we now send a response with body. We could change as follows:

- `Send a headers-only response` -> `Send an aggregated response`

Reply:
Moved to L53 so that it can be not Javadoc but just a comment.
Please take a look 😄